src/olympia/editors/templates/editors/review.html

```diff
@@ -116,6 +116,26 @@ <h3 id="history">
               {% set record = record_version.activity_log %}
               {% include 'editors/includes/history.html' %}
             {% endfor %}
+            {% if is_post_reviewer and version.is_ready_for_auto_approval %}
+              {# If it was not auto-approved, but could have been (i.e. it's a listed, unreviewed webextension update of a public extension) try to find out why. #}
+              <tr>
+                <th>{{ _('Auto Approval Summary') }}</th>
+                <td class="auto_approval">
+                  {% if version.autoapprovalsummary %}
+                    {# Call calculate_verdict() again, it will use the data already stored. Need to pass max_average_daily_users and min_approved_updates current values. #}
+                    {% set verdict_info=version.autoapprovalsummary.calculate_verdict(max_average_daily_users=auto_approval['max_average_daily_users'], min_approved_updates=auto_approval['min_approved_updates']) %}
+                    {% set verdict_info_human=version.autoapprovalsummary.verdict_info_prettifier(verdict_info) %}
```

Comment:
`version.autoapprovalsummary.calculate_verdict` should be called in `views.py` and passed in as a parameter, IMO.  If not the prettifier too (with _lazy changes)